spec/integration/add_to_rails_app_spec.rb

```diff
@@ -37,21 +38,28 @@ def run(command)
       "--no-rc",
       "--skip-bundle",
     ].join(" ")
-    FileUtils.chdir work_dir do
-      run rails_new
-      FileUtils.chdir rails_app_name do
-        example.run
+
+    # Use this local version of stitches rather than the one on Rubygems
+    gem_path = File.expand_path("../..", File.dirname(__FILE__))
+    use_local_stitches = %{echo "gem 'stitches', path: '#{gem_path}'" >> Gemfile}
+
+    Bundler.with_clean_env do
+      FileUtils.chdir work_dir do
+        run rails_new
+
+        FileUtils.chdir rails_app_name do
+          run use_local_stitches
+          run "gem install apitome responders rspec-rails rspec_api_documentation"
```

Comment:
This resolves an error on CI when running `bundle install`:

```
usr/local/lib/ruby/2.6.0/bundler/resolver.rb:287:in `block in verify_gemfile_dependencies_are_found!': Could not find gem 'apitome' in any of the gem sources listed in your Gemfile. (Bundler::GemNotFound)
```

I'm not a fan of this solution, so I'm open to ideas if anyone better understands what's going on.

Reply:
This stuck out to me as well.  Gregg and I talked about adding a code comment to try and explain why this is currently necessary.